docs/README.md

```diff
@@ -33,42 +37,736 @@ All output will adhere to the same general JSON structure:
+
+```json
+{
+  keys: [2, 15, 31],
+  values:  {
+    name: value
+  }
+}
+```
+
+::: danger WARNING
+Batch Update can quickly overwrite large amounts of data. Please be careful when implementing this request.
+:::
+
+#### Possible Results
+| Code                     | Description                                        |
+|--------------------------|----------------------------------------------------|
+| 200 OK                   | `data`: The updated item, including default fields added by Directus  |
+| 400 Bad Request          | `message`: Syntax error in provided JSON    |
+| 404 Not Found            | `message`: Collection doesn’t exist    |
+| 422 Unprocessable Entity | `message`: Column doesn’t exist in collection    |
+
+
+
+
+### Delete Item
+Deletes an item from a specific collection
+
+```http
+DELETE /items/[collection-name]/[primary-key-value]
+```
+
+#### Possible Results
+| Code                     | Description                                        |
+|--------------------------|----------------------------------------------------|
+| 204 No Content           | Record was successfully deleted        |
+| 404 Not Found            | `message`: Item doesn't exist within collection    |
+
+
+
+
+### Delete Items
+Deletes multiple items from a specific collection
+
+```http
+DELETE /items/[collection-name]/
+```
+
+#### Body
+An array of IDs of records to be deleted @TODO SHOULD THIS BE PKs?
+
+#### Possible Results
+| Code                     | Description                                        |
+|--------------------------|----------------------------------------------------|
+| 202 No Content @TODO 204?                  | Items were deleted successfully  |
+| 400 Bad Request          | `message`: Syntax error in provided JSON    |
+| 404 Not Found            | `message`: Collection doesn’t exist    |
+| 422 Unprocessable Entity | `message`: Items don't exist in collection    |
+
+
+
+
+
+
+## System
+
+All system tables (`directus_*`) are blocked from being used through the regular `/items` endpoint to prevent security leaks or because they require additional processing before sending to the end user. This means that any requests to `/items/directus_*` will always return `403 Forbidden`.
+
+These system endpoints still follow the same spec as a “regular” `/items/[collection-name]` endpoint but require the additional processing outlined below:
+
+### Columns
+`/columns` is used for creating, updating, or deleting columns through the API requires the API to modify the database schema directly. @TODO
+
+### Files
+`/files` is used for creating or updating a file requires the API to accept a special field allowing for the base64 file data. Beyond that, it must accept POST requests with the multipart-formdata enctype, to allow for easier uploading of file(s), and must accept uploading by chunk, to allow for larger filesizes. @TODO
+
+### Permissions
```

Comment:
Nothing really 😄 Just for consistencies sake with the other `directus_*` collections, it has it's own endpoint